wgpu-native/src/command/allocator.rs

```diff
@@ -7,10 +7,13 @@ use hal::pool::RawCommandPool;
 use hal::Device;
 use parking_lot::Mutex;
 
+use log::trace;
+
 use std::collections::HashMap;
 use std::sync::atomic::Ordering;
 use std::thread;
 
+
```

Comment:
I think rustfmt will keep resetting these double lines so it's probably best not to fight it :)

Reply:
ah, I want to configure that!